### DISCUSSION TITLE
FlightSQL support

### DISCUSSION THREAD

**Discussion by ahirner** [evidence-dev/evidence#2876] [Data Source Request]
### What do you use this database for?

I like to connect [FlightSQL](https://arrow.apache.org/docs/format/FlightSql.html) sources for [performance gains with analytics data](https://dev.to/apachedoris/arrow-flight-sql-in-apache-doris-for-10x-faster-data-transfermd-33ag).

Examples:
* https://github.com/spiceai/spiceai
* https://www.influxdata.com
* https://docs.dremio.com/current/sonar/developing-client-apps/arrow-flight-sql

### Are there any existing node clients for this data source?

Not a node dev. FlightSQL implementations are rooted in Rust and C++. Some searching turned up:
* https://github.com/roeap/flight-sql-client-node
* https://github.com/InfluxCommunity/influxdb3-js

### Are you interested in contributing this connector?

I'm just making a suggestion

**Comment by riziles** [comment]
FYI, the Lance DB team has one in beta under active development: https://github.com/lancedb/flight-sql-js-client 

BTW, Lance DB is awesome, and it would be great to have as a data source.
